es-vue-base/.eslintrc.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+    settings: {
+        'import/resolver': {
```

Comment:
I'm a little confused how this works in a rollup context.

Reply:
It doesn't. Each tool has their own alias resolver. This just tells `eslint` what `@` means in an import statement.